test/regress/bad-u64-leb128.txt

```diff
@@ -0,0 +1,25 @@
+;;; TOOL: run-interp-spec
+;;; ARGS: --enable-memory64
```

Comment:
Can we write this test without depending on `--enable-memory64`.

How about just a simple `i64.const` instruction?

Also, I guess this points to gap in the spec tests?

Reply:
> Can we write this test without depending on --enable-memory64

Unfortunately not, because the bug was in `ReadU64Leb128`. :-( Those are only used by memory64 (for memory min, memory max, and load/store offset -- but the first two are in units of pages, so limited to 2^48 anyway). The const literals are using signed Leb128s, e.g. `ReadS64Leb128`.

Reply:
Let me try to submit this to the memory64 proposal repo...

Reply:
OK.. maybe add a comment to that effect.

Reply:
Submitted PR here: https://github.com/WebAssembly/memory64/pull/38

Reply:
> OK.. maybe add a comment to that effect.

Done.